README.md

```diff
@@ -81,6 +114,9 @@ to both the current props and previous props. A common use case for this would b
   }
 ```
 
+In our theatre world, this could be a critic looking back at the play that was just performed and giving it a different
```

Comment:
'theatre' -> 'theater'

This is the only inconsistent usage of the spelling - let's stick with `theater` since our students will mostly be from the US anyway.